### DISCUSSION TITLE
Welcome to shapash Discussions!

### DISCUSSION THREAD

**Discussion by yg79** [MAIF/shapash#114] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by guptaarha** [comment]
Hi, This is really exciting way to understand and interpret black box models. Thank you very much. I was just wondering that is it possible to publish/deploy webapp on the server. We are currently using RStudio Teams which have python and R together in one server.  I can get the webapp for my model in local host but was interested in sharing the app with the team for brainstorming. If you can share the steps how we can publish this app then it would be great. Thank you once again.

**Reply by yg79** [reply]
Hello and thank you for your message, you can deploy your Shapash webapp and share it. This is what we did for our demonstrator https://shapash-demo.ossbymaif.fr.
Unlike R Shiny where you may have set up a shiny server to put your app.R files, it will be necessary here to put your app in a container (docker,... ) to make the app available. In this repo GitHub repo, you can see the code that launch our demonstrator: shapash/shapash/webapp/webapp_launch_DVF.py

**Reply by wahab4114** [reply]
Hi @yg79. I am also interested in doing the same. Can you elaborate more on how we can deploy the SHAPASH webapp in server/place within the company instead of individuals running it on their local machines?

**Reply by yg79** [reply]
Hello @wahab4114, sorry for my delay in response. We will write a post medium to explain how to do this.

**Reply by wahab4114** [reply]
@yg79 Thanks. Looking forward to it.

**Reply by esis21marta** [reply]
halo @yg79 i got the same problem like mr @wahab4114, when running run_app, the site cant be reached. i run via jupyter lab on vertex ai and already put an either external ip or internal internal ip address. do you know how to solve it?

<img width="650" alt="image" src="https://user-images.githubusercontent.com/36924845/190572552-6f0adb59-2db9-4415-aa49-9c8e4b3d2cbf.png">

<img width="634" alt="image" src="https://user-images.githubusercontent.com/36924845/190572657-6a8d1e98-6905-4bdf-a345-0ef49b0b63f9.png">


Thanks

**Comment by lokesh005** [comment]
@yg79, any update on how to deploy the SHAPASH webapp in server/place within the company instead of individuals running it on their local machines
